src/Repository/DefaultSluggableRepository.php

```diff
@@ -5,25 +5,29 @@
 namespace Knp\DoctrineBehaviors\Repository;
 
 use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ManagerRegistry;
 use Knp\DoctrineBehaviors\Contract\Entity\SluggableInterface;
 
 final class DefaultSluggableRepository
 {
     /**
-     * @var EntityManagerInterface
+     * @var ManagerRegistry
      */
-    private $entityManager;
+    private $managerRegistry;
 
-    public function __construct(EntityManagerInterface $entityManager)
+    public function __construct(ManagerRegistry $managerRegistry)
     {
-        $this->entityManager = $entityManager;
+        $this->managerRegistry = $managerRegistry;
     }
 
     public function isSlugUniqueFor(SluggableInterface $sluggable, string $uniqueSlug): bool
     {
         $entityClass = get_class($sluggable);
 
-        $queryBuilder = $this->entityManager->createQueryBuilder()
+        /** @var EntityManagerInterface $manager */
+        $manager = $this->managerRegistry->getManagerForClass($entityClass);
```

Comment:
```suggestion
        $entityManager = $this->managerRegistry->getManagerForClass($entityClass);
```

This would be more descriptive name :+1: